TPS_input.py

```diff
@@ -4,12 +4,12 @@
 import scipy as sp
 import numpy as np
 
-FULL_DATASET = False
+FULL_DATASET = True
 # Decide whether to run the old code with the full dataset or the new code
 # with selected peptides
 if (FULL_DATASET):
     # Load excel file of party processed data
-    data_xls = pd.ExcelFile('data/timeseries/merged_normalized.xlsx')
+    data_xls = pd.ExcelFile('/Users/jack/Downloads/tps-b1623110bd928e693ed33be5ffd8c71a83e3c1ff/data/timeseries/merged_normalized.xlsx')
```

Comment:
@JackRosenblatt1 can you please revert this change from your latest commit?  We'll need to use relative file paths so I can test it too.